src/test/uint256_tests.cpp

```diff
@@ -332,8 +332,15 @@ BOOST_AUTO_TEST_CASE(parse)
 
 BOOST_AUTO_TEST_CASE( check_ONE )
 {
-    uint256 one = uint256S("0000000000000000000000000000000000000000000000000000000000000001");
+    uint256 one = uint256{"0000000000000000000000000000000000000000000000000000000000000001"};
     BOOST_CHECK_EQUAL(one, uint256::ONE);
 }
 
+BOOST_AUTO_TEST_CASE(FromHex_vs_uint256)
+{
+    auto runtime_uint{uint256::FromHex("4A5E1E4BAAB89F3A32518A88C31BC87F618f76673e2cc77ab2127b7afdeda33b").value()};
```

Comment:
are we sure we want to mandate that mixed-case hex strings can be parsed?

Reply:
Thanks for considering, I'm personally fine with doing it in a different PR as well

Reply:
I don't have a strong view on this but I would prefer getting this PR over the line as is.

Reply:
Trying to not spawn yet another PR from this work but will post that as a separate PR then if this one is merged in the current state.